integreat_cms/core/settings.py

```diff
@@ -1005,6 +1005,11 @@
     strtobool(os.environ.get("INTEGREAT_CMS_SUMM_AI_IS_INITIAL", "True"))
 )
 
+# Slugs of regions that prefer Plain German over Easy German
+SUMM_AI_PLAIN_GERMAN_REGIONS: Final[str | list[str] | None] = os.environ.get(
+    "INTEGREAT_CMS_SUMM_AI_PLAIN_GERMAN_REGIONS", []
+)
```

Comment:
So how does this translate into specifying the environment variable?
```sh
$ INTEGREAT_CMS_SUMM_AI_PLAIN_GERMAN_REGIONS=augsburg, testregion
bash: testregion: command not found
$ INTEGREAT_CMS_SUMM_AI_PLAIN_GERMAN_REGIONS=augsburg,testregion python
Python 3.11.8 (main, Feb 12 2024, 14:50:05) [GCC 13.2.1 20230801] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.environ.get("INTEGREAT_CMS_SUMM_AI_PLAIN_GERMAN_REGIONS", [])
'augsburg,testregion'
>>> 
```
Well, we need to do an additional step here. Environmental variables will only be strings, but luckily we're using slugs that won't contain special characters like, say, commas, so we can use them as separators:
```suggestion
# Slugs of regions that prefer Plain German over Easy German
SUMM_AI_PLAIN_GERMAN_REGIONS: Final[str | list[str] | None] = [x.strip() for x in os.environ.get(
    "INTEGREAT_CMS_SUMM_AI_PLAIN_GERMAN_REGIONS", ""
).split(",")]
```

Reply:
🤔? I've imagined something like SUMM_AI_PLAIN_GERMAN_REGIONS = ["augsburg", "nurnberg"]

Reply:
This works in python, but not in bash or anywhere you would set environment variables

Reply:
Oh wait, I was hasty. `splitlines()` is used at a few places in the settings.py. Better be consistent, even if I wasn't sure whether it was trivial making environment variables contain newlines.
```suggestion
# Slugs of regions that prefer Plain German over Easy German
SUMM_AI_PLAIN_GERMAN_REGIONS: Final[str | list[str] | None] = [
    x.strip()
    for x in os.environ.get(
        "INTEGREAT_CMS_SUMM_AI_PLAIN_GERMAN_REGIONS", ""
    ).splitlines()
]
```